### PR TITLE
FAAS-431 - Add confirmation page details.  Update form card backgroun…

### PR DESCRIPTION
**Pull request by stephanie-graham**
…d color.

### CODE REVIEW

File: src/main/webapp/app/modules/home/home.scss [hunk 1]
```diff
@@ -30,3 +30,8 @@ ul.custom {
     width: 100%;
   }
 }
+
+a {
+  color: #005ea2;
+  font-weight: bold;
+}
```
Comment (Wube-T-Kifle) [inline src/main/webapp/app/modules/home/home.scss]: Move this CSS to it is own confirmation.scss file

File: src/main/webapp/app/modules/confirmation/confirmation.tsx [hunk 2]
```diff
@@ -1,22 +1,94 @@
+import '../home/home.scss';
 import React, { useCallback } from 'react';
 import { Link, useHistory } from 'react-router-dom';
 import { Form } from 'react-formio';
 import uswds from '@formio/uswds';
 import { Formio } from 'formiojs';
```
Comment (Wube-T-Kifle) [inline src/main/webapp/app/modules/confirmation/confirmation.tsx]: remove unused imports

File: src/main/webapp/app/modules/confirmation/confirmation.tsx [hunk 3]
```diff
@@ -1,22 +1,94 @@
+import '../home/home.scss';
```
Comment (Wube-T-Kifle) [inline src/main/webapp/app/modules/confirmation/confirmation.tsx]: Move this CSS to it is own confirmation.scss file

File: src/main/webapp/app/modules/home/home.scss (hunk 1 above)
Comment (stephanie-graham) [reply src/main/webapp/app/modules/home/home.scss]: Done.

File: src/main/webapp/app/modules/confirmation/confirmation.tsx (hunk 2 above)
Comment (stephanie-graham) [reply src/main/webapp/app/modules/confirmation/confirmation.tsx]: You did this, thank you.

File: src/main/webapp/app/modules/confirmation/confirmation.tsx (hunk 3 above)
Comment (stephanie-graham) [reply src/main/webapp/app/modules/confirmation/confirmation.tsx]: You did this, thank you!

**Pull request merged**